frontend/src/routes/(app)/(internal)/entity-assessments/[id=uuid]/+page.server.ts

```diff
@@ -1,7 +1,49 @@
+import { BASE_API_URL } from '$lib/utils/constants';
 import { getModelInfo } from '$lib/utils/crud';
 import { loadDetail } from '$lib/utils/load';
-import type { PageServerLoad } from './$types';
+import { fail, superValidate } from 'sveltekit-superforms';
+import type { Actions, PageServerLoad } from './$types';
+import { z } from 'zod';
+import { zod } from 'sveltekit-superforms/adapters';
+import * as m from '$paraglide/messages';
+import { setFlash } from 'sveltekit-flash-message/server';
+import { safeTranslate } from '$lib/utils/i18n';
 
 export const load: PageServerLoad = async (event) => {
 	return loadDetail({ event, model: getModelInfo('entity-assessments'), id: event.params.id });
 };
+
+export const actions: Actions = {
+	mailing: async ({ request, fetch, cookies }) => {
```

Comment:
`mailing` is not a very explicit name. Would we not gain better readability if we actually stated what this action does? E.g. `emailQuestionnaire`...
The URL of the endpoint should also be more specific than `/entity-assessments/mailing`

Reply:
Yes I agree for the name, we'll need to change it anyway to add another action to send the external questionnaire by mail.
However, the endpoint is `/compliance-assessments/mailing` because we send the audit and not the entity-assessment. It is not explicit because it could be useful to send other audits than questionnaires in the future.

Reply:
Was a typo, meant compliance-assessment/mailing :)
The point is that the word 'mailing' does not explicitly convey what the endpoint actually does